components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Publisher/src/Services/PublisherJobService.cs

```diff
@@ -229,6 +229,13 @@ public sealed class PublisherJobService : IPublishServices<string> {
                 var publishJob = (WriterGroupJobModel)_serializer.DeserializeJobConfiguration(
                     job.JobConfiguration, job.JobConfigurationType);
                 if (publishJob != null) {
+
+                    if (publishJob.Engine != null) {
+                        publishJob.Engine.BatchTriggerInterval = null;
```

Comment:
why is this needed?

Reply:
The goal is that any update to existing jobs will write the job back to the database with the trigger empty and batchsize 1, migrating existing jobs as they are upgraded.   It is layer 2 of the 3 layer safety net:  1. create new jobs without batching, 2. update existing jobs as they are updated, 3. disregard any batch size or interval in publisher 2.7.171.   The 3rd step is needed if 1 or 2 are not happening.